yb-voyager/cmd/templates/migration_assessment_report.template

```diff
@@ -11,7 +11,7 @@
             color: #333;
         }
         .wrapper {
-            max-width: 960px;
+            max-width: 70%;
```

Comment:
would this be centered? from the screenshot you shared on the PR desc, it doesn't look like it?

Reply:
yes it is centered, screenshot is only some part of screen only that's why it doesn't look center. will update.